mytb/tests/test_path.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import absolute_import
+
+import os
+
+import pytest
+
+from mytb.exceptions import MyTBError
+from mytb.path import robust_makedirs
+
+
+def test_robust_makedirs(tmp_path, monkeypatch):
+
+    # create new dir and check no fail if existing
+    dir1 = str(tmp_path / "d1" / "d2")
```

Comment:
Why don't use os.path.join ?

Reply:
This is no string concatenation. This is the new pathlib ( https://docs.python.org/3/library/pathlib.html ) where elements can be joined safely with the `/` operator

`pathlib` is the _newer_ way of working with paths and file names